File: CS/CustomLanguageResource/MauiProgram.cs
using System.Globalization;
using Microsoft.Maui;
using Microsoft.Maui.Hosting;
using Microsoft.Maui.Controls.Hosting;
using DevExpress.Maui;

namespace CustomLanguageResource;

public static class MauiProgram
{
    public static MauiApp CreateMauiApp()
    {
        Thread.CurrentThread.CurrentUICulture = Thread.CurrentThread.CurrentCulture = new CultureInfo("pt-BR");
        var builder = MauiApp.CreateBuilder();
        builder
            .UseDevExpress()
            .UseDevExpressCollectionView()
            .UseDevExpressControls()
            .UseDevExpressEditors()
            .UseDevExpressScheduler()
            .UseMauiApp<App>()
            .ConfigureFonts(fonts =>
            {
                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
                fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
            });
        return builder.Build();
    }
}

File: VB/CustomLanguageResource/MauiProgram.vb
Imports System.Globalization
Imports Microsoft.Maui
Imports Microsoft.Maui.Hosting
Imports Microsoft.Maui.Controls.Hosting
Imports DevExpress.Maui

Namespace CustomLanguageResource

    Public Module MauiProgram

        Public Function CreateMauiApp() As MauiApp
            Thread.CurrentThread.CurrentCulture = New CultureInfo("pt-BR")
            Thread.CurrentThread.CurrentUICulture = Thread.CurrentThread.CurrentCulture
            Dim builder = MauiApp.CreateBuilder()
            builder.UseDevExpress().UseMauiApp(Of App)().ConfigureFonts(Function(fonts)
                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular")
                fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold")
            End Function)
            Return builder.Build()
        End Function
    End Module
End Namespace
